property/collector.go

```diff
@@ -96,7 +124,27 @@ func (p *Collector) CreateFilter(ctx context.Context, req types.CreateFilter) er
 	return nil
 }
 
-func (p *Collector) WaitForUpdates(ctx context.Context, version string, opts ...*types.WaitOptions) (*types.UpdateSet, error) {
+func (p *Collector) CreateFilterEx(ctx context.Context, req types.CreateFilter) (*Filter, error) {
```

Comment:
couple of breaking changes already, how about changing the existing `CreateFilter` to return `(*Filter, error)` instead of adding `CreateFilterEx` ?